### PR TITLE
Fix/cb-5019/components unification

### PR DESCRIPTION
**Pull request by Wroud**

### CODE REVIEW

**Review by devnaumov — APPROVED** [review]

File: webapp/packages/core-blocks/src/FormControls/Filter.m.css [hunk 1]
```diff
@@ -1,39 +1,47 @@
...
-  cursor: auto;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  top: 50%;
+  right: 2px;
+  transform: translateY(-50%);
+}
+
+.actionButton {
```
Comment (sergeyteleshev) [inline webapp/packages/core-blocks/src/FormControls/Filter.m.css:29]: empty class

File: webapp/packages/core-blocks/src/FormControls/Filter.tsx [hunk 2]
```diff
@@ -47,89 +50,92 @@ export const Filter = observer<ControlledProps | ObjectsProps<any, any>>(functio
...
-
-      if (onChange) {
-        onChange(value, name);
-      }
-    },
-    [onChange, state],
-  );
-
+  let valuePassedFromProps = isNotNullDefined(valueControlled);
   let value: any = valueControlled;
```
Comment (sergeyteleshev) [inline webapp/packages/core-blocks/src/FormControls/Filter.tsx:65]: please typescript this variable

File: webapp/packages/core-blocks/src/FormControls/Filter.tsx [hunk 3]
```diff
@@ -47,89 +50,92 @@ export const Filter = observer<ControlledProps | ObjectsProps<any, any>>(functio
...
         type="search"
-        className={s(styles, { inputField: true, max })}
+        autoComplete="off"
+        className={s(styles, { inputField: true })}
         placeholder={placeholder}
         disabled={disabled}
         name={name}
         value={value}
-        onChange={filter}
+        onChange={handleChange}
```
Comment (sergeyteleshev) [inline webapp/packages/core-blocks/src/FormControls/Filter.tsx:122]: I believe we need a debounce here with 100-150ms

**Review by sergeyteleshev — APPROVED** [review]

File: webapp/packages/core-blocks/src/FormControls/Filter.m.css (hunk 1 above)
Comment (Wroud) [reply webapp/packages/core-blocks/src/FormControls/Filter.m.css:29]: it's on purpose, later you can use register to add styles for elements with this class

**Pull request merged**